packages/strapi-parts/src/Radio/Radio.stories.mdx

```diff
@@ -35,3 +49,8 @@ Description...
     }}
   </Story>
 </Canvas>
+
+## Props
+
+<ArgsTable of={Radio} />
```

Comment:
We might need to specify the propTypes for the Radio component like the `name` and the `onChange`.

Reply:
Also it could be great if we could show how we import the Radio component:

```
import { Radio } from '@strapi/parts/Radio';
```